resources/docker-images/build.sh

```diff
@@ -75,7 +80,7 @@ build_action() {
   _convert_parameters_to_build_args
 
   if [[ "${platform}" == "base" ]]; then
-    docker pull --platform "amd64" "ubuntu:${UBUNTU_VERSION:-${KEYMAN_DEFAULT_VERSION_UBUNTU_CONTAINER}}"
+    docker pull --platform "amd64" "${DISTRO:-ubuntu}:${DISTRO_VERSION:-${KEYMAN_DEFAULT_VERSION_UBUNTU_CONTAINER}}"
```

Comment:
```suggestion
    docker pull --platform "amd64" "${DISTRO}:${DISTRO_VERSION}"
```

Do we need the defaults here again? I see e.g. on l.111 we don't refer to the defaults, and it seems like `_convert_parameters_to_build_args` should be responsible for setting defaults too, although I am not 100% clear on what that is doing.

Reply:
`_convert_parameters_to_build_args` did not set the defaults, but then we'd have to have the defaults further down again (which I had missed). 

It makes sense that `_convert_parameters_to_build_args` would set the defaults, so I modified it to that. Thanks!

Reply:
Cool ... unfortunately I think that means `_is_default_values` will no longer work?

Reply:
You're right, I need to do that differently as well. Thanks!